legal/satzung.md

```diff
@@ -0,0 +1,123 @@
+
+
+## Organe des Vereins
+
+1. Die Organe des Vereins sind
+    1. die Mitgliederversammlung und
+    2. der Vorstand.
+2. Der Vorstand besteht aus drei Mitgliedern,
+    1. der Sprecher*in,
+    2. der Schriftführer*in und
+    3. der Kassenwart*in.
+    Diese dürfen nicht beim selben Arbeitgeber beschäftigt sein.
+3. Der Vorstand vertritt den Verein gerichtlich und außergerichtlich im Sinne des § 26 BGB,
+    wobei jedes Vorstandsmitglied für sich zur Vertetung des Vereins berechtigt ist.
+4. Der Vorstand gibt sich eine Geschäftsordnung, die von der Mitgliederversammlung bestätigt werden muss.
+
+
+## Treffen und Kommunikation
+
+1. Sämtliche Treffen des Vereins und seiner Organe können mittels Fernkommunikationsmittel unter Beachtung zumutbarer Identitätskontrollen durchgeführt werden.
+2. Benachrichtigungsfristen gelten als erfüllt, wenn die Benachrichtigung vor Verstreichen der Frist abgeschickt oder veröffentlicht wurde.
+3. Eine Benachrichtigung gilt als zugestellt, wenn sie an die letzte dem Verein bekannte Adresse verschickt oder auf der Webseite des Vereins veröffentlicht wurde.
+4. Bei Zustimmung einer zu benachrichtigenden Person kann gegenüber dieser auf die Einhaltung von Benachrichtigungsfristen und -formalien verzichtet werden.
+
+
+## Mitgliedschaft
+
+1. Mitglieder des Vereins können volljährige natürliche sowie juristische Personen werden.
+2. Die Beitrittserklärung erfolgt gegenüber dem Vorstand.
+3. Über Aufnahme entscheidet der Vorstand. Die Aufnahme erfolgt nach freiem Ermessen.
+    Bei Ablehnung ist er nicht verpflichtet, dem Antragsteller die Gründe hierfür mitzuteilen.
+4. Die Mitgliedschaft endet durch Austritt aus dem Verein, durch Tod, Ausschluss, Streichung von der Mitgliederliste.
+5. Ein Mitglied kann durch Beschluss des Vorstandes von der Mitgliederliste gestrichen werden, wenn es trotz schriftlicher Mahnung mit der Bezahlung von Mitgliedsbeiträgen oder von Umlagen im Rückstand ist.
+    Diese Streichung befreit das Mitglied nicht von der Begleichung rückständiger Beiträge und Umlagen.
+6. Ein Mitglied kann ausgeschlossen werden, wenn es schuldhaft in grober Weise die Interessen des Vereins verletzt. Der Ausschluss erfolgt durch Beschluss [des Vorstandes / der Mitgliederversammlung]: Vor der Beschlussfassung muss dem Mitglied Gelegenheit zur mündlichen oder schriftlichen Stellungnahme gegeben werden. Der Beschluss ist schriftlich zu begründen und dem Mitglied mitzuteilen. Hiergegen kann innerhalb eines Monats nach Zugang des Beschlusses beim Vorstand Widerspruch eingelegt werden. Über diesen Widerspruch entscheidet die nächste Mitgliederversammlung endgültig.
+
+
+## Beiträge
+
+1. Der Verein kann einen Jahresbeitrag und einen Aufnahmebeitrag erheben.
+    Die maximale Höhe dieser Beiträge ist von der Mitgliederversammlung festzulegen.
+2. In begründeten Fällen können Mitglieder von der Beitragspflicht ausgenommen werden.
+3. Näheres regelt die Beitragsordnung.
+
+
+## Mitgliederversammlung
```

Comment:
Do we want to allow persons having two votes (if an institution assigns vote to a person who is personally member as well)?
yes:
 * makes it less complicated for institutes to vote and the vote needs to be given to somebody which is not that much involved

no:
 * effectively, single people have more weight

Reply:
At the meeting of 22-03, there was consensus to keep it like it is, that means a person can have two votes (one, if he/she is member, and the second, if he/she additionally has the voting rights of his institution.